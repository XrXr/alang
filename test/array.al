main :: proc() {
    var array [5]int
    array[0]   = 100
    array[1]   = 25
    array[2]   = 2
    array[3]   = 3
    array[2+2] = 1

    ret := array[0]
    ret = ret / array[1]
    ret = ret / array[2]
    ret = ret * array[3]
    ret = ret - array[4]
    puts("should be 5\n")
    print_int(ret)
}
