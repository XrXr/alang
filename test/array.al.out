should be 5
5
